modules/cloud-function/variables.tf

```diff
@@ -50,13 +50,13 @@ variable "environment_variables" {
 }
 
 variable "function_config" {
-  description = "Cloud function configuration."
+  description = "Cloud function configuration. Defaults to using main as entrypoint, 1 instance with 256MiB of memory, and 180 second timeout"
   type = object({
-    entry_point = string
-    instances   = number
-    memory      = number # Memory in MB
-    runtime     = string
-    timeout     = number
+    entry_point = optional(string, "main")
+    instances   = optional(number, 1)
+    memory      = optional(number, 256) # Memory in MB
+    runtime     = optional(string, "python37")
```

Comment:
Can we default to python310 which is the current [recommended runtime for python](https://cloud.google.com/functions/docs/concepts/python-runtime)?

Reply:
Happy to update that in 2f71b68db327ac26dc3248ab705b537f2b2f4bb1